File: Java/FilmBioskop.java
public class FilmBioskop extends Film {
    private int durasi;
    private String studio;
    private String klasifikasiUsia;

    // Constructor default
    public FilmBioskop() {
        super();
        this.durasi = 0;
        this.studio = "";
        this.klasifikasiUsia = "";
    }

    // Constructor berparameter
    public FilmBioskop(
        int id,
        String judul,
        String genre,
        int durasi,
        String studio,
        String klasifikasiUsia
    ) {
        super(id, judul, genre);
        this.durasi = durasi;
        this.studio = studio;
        this.klasifikasiUsia = klasifikasiUsia;
    }

    // Getter
    public int getDurasi() { return durasi; }
    public String getStudio() { return studio; }
    public String getKlasifikasiUsia() { return klasifikasiUsia; }

    // Setter
    public void setDurasi(int durasi) { this.durasi = durasi; }
    public void setStudio(String studio) { this.studio = studio; }
    public void setKlasifikasiUsia(String klasifikasiUsia) { this.klasifikasiUsia = klasifikasiUsia; }
}

File: PHP/FilmBioskop.php
<?php

require_once "Film.php";

class FilmBioskop extends Film {
    private int $durasi;
    private string $studio;
    private string $klasifikasiUsia;

    // Constructor default / berparameter
    public function __construct(
        int $id = 0,
        string $judul = "",
        string $genre = "",
        int $durasi = 0,
        string $studio = "",
        string $klasifikasiUsia = ""
    ) {
        parent::__construct($id, $judul, $genre);
        $this->durasi          = $durasi;
        $this->studio          = $studio;
        $this->klasifikasiUsia = $klasifikasiUsia;
    }

    // Getter
    public function getDurasi(): int             { return $this->durasi; }
    public function getStudio(): string          { return $this->studio; }
    public function getKlasifikasiUsia(): string { return $this->klasifikasiUsia; }

    // Setter
    public function setDurasi(int $durasi): void                        { $this->durasi = $durasi; }
    public function setStudio(string $studio): void                     { $this->studio = $studio; }
    public function setKlasifikasiUsia(string $klasifikasiUsia): void   { $this->klasifikasiUsia = $klasifikasiUsia; }
}
